hardware_integration/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ set(HARDWARE_INTEGRATION_SRC_DIR "src")
 set(HARDWARE_INTEGRATION_INCLUDE_DIR "include/isobus/hardware_integration")
 
 if(NOT CAN_DRIVER)
+  message(AUTHOR_WARNING "No CAN driver specified, choosing Linux socket CAN by default. Use -DCAN_DRIVER=\"SocketCAN\" to specify another driver. or -DCAN_DRIVER=\"SocketCan;MCP2515\" to specify multiple drivers.")
```

Comment:
Duplicate message to line 13, probably just missed it when rebasing.  I would suggest combining the two into:
```
"No CAN driver specified, choosing Linux socket CAN by default. Set the CAN_DRIVER variable with -DCAN_DRIVER=<driver> to specify another driver. or -DCAN_DRIVER=\"SocketCAN;MCP2515;WindowsPCANBasic\" to specify multiple drivers."
```

Reply:
I see, I changed your suggestion a bit to make it even more abstract, let me know what you think:
```cmake
if(NOT CAN_DRIVER)
  if (WIN32)
    set(CAN_DRIVER "WindowsPCANBasic")
  else()
    set(CAN_DRIVER "SocketCAN")
  endif()

  message(
    AUTHOR_WARNING
    "No CAN driver specified, choosing ${CAN_DRIVER} by default. Set the CAN_DRIVER variable with -DCAN_DRIVER=<driver> to specify another driver. or -DCAN_DRIVER=\"<driver1>;<driver2>\" to specify multiple drivers."
  )
endif()
```

Reply:
I guess we also need some documentation soon on what drivers are available with the growing number of them

Reply:
Added relevant information to `README.md` in commit https://github.com/ad3154/ISO11783-CAN-Stack/pull/91/commits/7e7b48ec7bb22af825ca84740fb4d0caef7d3a2e

Reply:
Looks good to me!